tests/phpunit.xml

```diff
@@ -10,6 +10,7 @@
       <file>backdrop/DownloadCommandsTest.php</file>
       <file>backdrop/HelpCommandsTest.php</file>
       <file>backdrop/PHPCommandsTest.php</file>
+      <file>backdrop/ScriptCommandsTest.php</file>
```

Comment:
The convention is for alphabetical order; I appreciate you put it there as its closely related to the previous command.  If you accept my wider suggestion then won't be needed.